src/test/java/com/google/sps/CommentRetrievalTest.java

```diff
@@ -45,6 +45,8 @@
 
   // Simulate a CommentThreadListResponse with exactly as many comments as expected
   public void setUpYouTubeMocks(int numExpectedComments) throws Exception {
+    // For the API Key
+    System.setProperty("resources-folder", "/src/main/webapp");
```

Comment:
Question: Is this needed for the test to pass?

Reply:
Yes, it is. This is what lets it access the API Key in the /webapp folder.